[2021-12-14 02:24:31,456] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_dublin.scrap_data scheduled__2021-12-13T01:30:00+00:00 [queued]>
[2021-12-14 02:24:31,875] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_dublin.scrap_data scheduled__2021-12-13T01:30:00+00:00 [queued]>
[2021-12-14 02:24:31,881] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-14 02:24:31,888] {taskinstance.py:1242} INFO - Starting attempt 2 of 4
[2021-12-14 02:24:31,894] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-14 02:24:32,275] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-13 01:30:00+00:00
[2021-12-14 02:24:32,809] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_dublin', 'scrap_data', 'scheduled__2021-12-13T01:30:00+00:00', '--job-id', '2066', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_dublin.py', '--cfg-path', '/tmp/tmpaj2qm0i8', '--error-file', '/tmp/tmpabd3wqdv']
[2021-12-14 02:24:32,871] {standard_task_runner.py:77} INFO - Job 2066: Subtask scrap_data
[2021-12-14 02:24:32,548] {standard_task_runner.py:52} INFO - Started process 31519 to run task
[2021-12-14 02:24:35,133] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_dublin.scrap_data scheduled__2021-12-13T01:30:00+00:00 [running]> on host c81f81825bc0
[2021-12-14 02:24:39,357] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_dublin
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-13T01:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-13T01:30:00+00:00
[2021-12-14 02:24:40,186] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-14 02:24:40,377] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py dublin
[2021-12-14 02:25:42,371] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-14 02:25:48,323] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-14 02:25:48,335] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-14 02:25:48,345] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-14 02:25:48,357] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-14 02:25:48,377] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-14 02:25:52,119] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-14 02:25:52,124] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-14 02:25:52,133] {spark_submit.py:514} INFO - deployMode              null
[2021-12-14 02:25:52,143] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-14 02:25:52,150] {spark_submit.py:514} INFO - executorCores           null
[2021-12-14 02:25:52,155] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-14 02:25:52,163] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-14 02:25:52,179] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-14 02:25:52,198] {spark_submit.py:514} INFO - driverCores             null
[2021-12-14 02:25:52,209] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-14 02:25:52,217] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-14 02:25:52,244] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-14 02:25:52,259] {spark_submit.py:514} INFO - supervise               false
[2021-12-14 02:25:52,274] {spark_submit.py:514} INFO - queue                   null
[2021-12-14 02:25:52,281] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-14 02:25:52,291] {spark_submit.py:514} INFO - files                   null
[2021-12-14 02:25:52,301] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-14 02:25:52,309] {spark_submit.py:514} INFO - archives                null
[2021-12-14 02:25:52,316] {spark_submit.py:514} INFO - mainClass               null
[2021-12-14 02:25:52,328] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-14 02:25:52,334] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-14 02:25:52,359] {spark_submit.py:514} INFO - childArgs               [dublin]
[2021-12-14 02:25:52,371] {spark_submit.py:514} INFO - jars                    null
[2021-12-14 02:25:52,379] {spark_submit.py:514} INFO - packages                null
[2021-12-14 02:25:52,384] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-14 02:25:52,392] {spark_submit.py:514} INFO - repositories            null
[2021-12-14 02:25:52,413] {spark_submit.py:514} INFO - verbose                 true
[2021-12-14 02:25:52,423] {spark_submit.py:514} INFO - 
[2021-12-14 02:25:52,430] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-14 02:25:52,441] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-14 02:25:52,460] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-14 02:25:52,472] {spark_submit.py:514} INFO - 
[2021-12-14 02:25:52,485] {spark_submit.py:514} INFO - 
[2021-12-14 02:41:16,553] {spark_submit.py:514} INFO - Main class:
[2021-12-14 02:41:16,558] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-14 02:41:16,566] {spark_submit.py:514} INFO - Arguments:
[2021-12-14 02:41:16,571] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-14 02:41:16,575] {spark_submit.py:514} INFO - null
[2021-12-14 02:41:16,579] {spark_submit.py:514} INFO - dublin
[2021-12-14 02:41:16,582] {spark_submit.py:514} INFO - --verbose
[2021-12-14 02:41:16,588] {spark_submit.py:514} INFO - Spark config:
[2021-12-14 02:41:16,593] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-14 02:41:16,597] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-14 02:41:16,604] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-14 02:41:16,610] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-14 02:41:16,615] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-14 02:41:16,617] {spark_submit.py:514} INFO - 
[2021-12-14 02:41:16,625] {spark_submit.py:514} INFO - 
[2021-12-14 02:41:16,626] {spark_submit.py:514} INFO - 
[2021-12-14 02:56:39,666] {local_task_job.py:212} WARNING - State of this instance has been externally set to up_for_retry. Terminating instance.
[2021-12-14 02:56:39,867] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 31519
[2021-12-14 02:56:39,908] {taskinstance.py:1411} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-12-14 02:56:39,949] {spark_submit.py:644} INFO - Sending kill signal to spark-submit
[2021-12-14 02:56:43,150] {process_utils.py:66} INFO - Process psutil.Process(pid=31859, status='terminated', started='02:26:20') (31859) terminated with exit code None
[2021-12-14 02:56:44,782] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 433, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 482, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1413, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2021-12-14 02:56:45,462] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_dublin, task_id=scrap_data, execution_date=20211213T013000, start_date=20211214T022431, end_date=20211214T025645
[2021-12-14 02:56:46,370] {standard_task_runner.py:91} ERROR - Failed to execute job 2066 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 433, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 482, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1413, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2021-12-14 02:56:47,436] {process_utils.py:66} INFO - Process psutil.Process(pid=31519, status='terminated', exitcode=1, started='02:24:32') (31519) terminated with exit code 1
[2021-12-14 02:56:47,452] {process_utils.py:66} INFO - Process psutil.Process(pid=31548, status='terminated', started='02:24:40') (31548) terminated with exit code None
